app/src/main/java/com/android/streetworkapp/ui/park/EventOverview.kt

```diff
@@ -0,0 +1,252 @@
+          Column(modifier = Modifier.padding(padding).fillMaxHeight()) {
+            Row(
+                modifier = Modifier.padding(16.dp),
+                verticalAlignment = Alignment.CenterVertically) {
+                  Icon(
+                      Icons.Outlined.AccountCircle,
+                      contentDescription = "User",
+                      modifier = Modifier.padding(horizontal = 8.dp).testTag("ownerIcon"))
+                  Text("Organized by: ${event.owner}", modifier = Modifier.testTag("eventOwner"))
+                }
+
+            Row(
+                modifier = Modifier.padding(16.dp),
+                verticalAlignment = Alignment.CenterVertically) {
+                  Icon(
+                      Icons.Filled.DateRange,
+                      contentDescription = "Date",
+                      modifier = Modifier.padding(horizontal = 8.dp).testTag("dateIcon"))
+                  Text(event.date.toFormattedString(), modifier = Modifier.testTag("date"))
+                }
+
+            Row(
+                modifier = Modifier.padding(16.dp),
+                verticalAlignment = Alignment.CenterVertically) {
+                  Icon(
+                      Icons.TwoTone.Face,
+                      contentDescription = "participants",
+                      modifier = Modifier.padding(horizontal = 8.dp).testTag("participantsIcon"))
+                  Text(
+                      "Participants: ${event.participants}/${event.maxParticipants}",
+                      modifier = Modifier.testTag("participants"))
+                }
+
+            Spacer(modifier = Modifier.height(16.dp))
+
+            EventDashboard(event)
+
+            val parkLatLng = LatLng(park.location.lat, park.location.lon)
+
+            // Create a CameraPositionState to control the camera position
+
+            val cameraPositionState = rememberCameraPositionState {
+              position = CameraPosition.Builder().target(parkLatLng).zoom(15f).build()
+            }
+
+            Row(
+                modifier = Modifier.padding(bottom = 16.dp),
+                verticalAlignment = Alignment.CenterVertically) {
+                  Icon(
+                      Icons.Outlined.LocationOn,
+                      contentDescription = "location",
+                      modifier = Modifier.padding(horizontal = 8.dp).testTag("locationIcon"))
+                  Text("at ${park.name}", modifier = Modifier.testTag("location"))
+                }
+
+            GoogleMap(
+                modifier = Modifier.testTag("googleMap").fillMaxWidth().height(300.dp),
+                cameraPositionState = cameraPositionState) {
+                  Marker(
+                      contentDescription = "Marker",
+                      state =
+                          MarkerState(position = LatLng(parkLatLng.latitude, parkLatLng.longitude)))
+                }
+          }
+        }
+      }
+}
```

Comment:
Improvement:
As the [coach](https://github.com/SwEnt-Group8/Street-work-app/pull/23#discussion_r1797771765) (@Andy130604) says, we need to limit the size of our composables. 

In your case, I'd divide this big composable like this (you can do it differently, but I think it's better to divide it)

```kotlin
fun EventTopBar(navigationActions: NavigationActions, title: String) {
fun EventBottomBar(event: Event) {
fun EventDetails(event: Event) {
fun EventMap(park: Park) {
fun EventOverviewScreen(navigationActions: NavigationActions, event: Event, park: Park) { //As the main Composable
```